src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/IDesignerSerializationManager.cs

```diff
@@ -71,30 +71,30 @@ public interface IDesignerSerializationManager : IServiceProvider
         /// of named instances. Objects that implement IComponent will be
         /// added to the design time container if addToContainer is true.
         /// </summary>
-        object CreateInstance(Type type, ICollection arguments, string name, bool addToContainer);
+        object CreateInstance(Type type, ICollection arguments, string? name, bool addToContainer);
 
         /// <summary>
         /// Retrieves an instance of a created object of the given name, or
         /// null if that object does not exist.
         /// </summary>
-        object GetInstance(string name);
+        object? GetInstance(string name);
 
         /// <summary>
         /// Retrieves a name for the specified object, or null if the object
         /// has no name.
         /// </summary>
-        string GetName(object value);
+        string? GetName(object value);
```

Comment:
Same here:
https://github.com/dotnet/winforms/blob/00e6b62faf223b3cf2c2417b888de71c1648fbee/src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/DesignerSerializationManager.cs#L779-L782

Reply:
Same there, it's validating that value is _not_ null.